plugin/general/editor.go

```diff
@@ -23,12 +23,13 @@ import (
 
 	"fmt"
 
+	"context"
```

Comment:
It might be better to group all imports of standard library together so `go fmt` can sorts import by their name properly

https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING.md#imports-grouping

Reply:
Thanks, will group imports properly in next pr.